Framework/PythonInterface/mantid/kernel/src/Exports/ConfigService.cpp

```diff
@@ -50,6 +50,16 @@ const InstrumentInfo &getInstrument(ConfigServiceImpl &self, const object &name
     return self.getInstrument(ExtractStdString(name)());
 }
 
+/// duck typing emulating dict.get method
+std::string getStringUsingCacheElseDefault(ConfigServiceImpl &self, const std::string &key,
+                                           const std::string &defaultValue) {
+  std::vector<std::string> keys = self.keys();
+  if (std::find(keys.begin(), keys.end(), key) != keys.end())
```

Comment:
Use the existing function
```suggestion
  if (hasProperty(key))
```

Reply:
good catch! :smile: